packages/UI/tsconfig.json

```diff
@@ -10,6 +10,7 @@
     "noEmit": false,
     "composite": false,
     "incremental": true,
+    "outDir": "dist",
```

Comment:
@Enjoy2Live do we have things that depend on this path? Or does this only matter for local development?

Reply:
I requested a revert of this and just got merged